concepticondata/conceptlists/Araujo-1996-289.tsv

```diff
@@ -0,0 +1,290 @@
+Araujo-1996-289-122	122	Old Person	725.0	OLD PERSON	Alter, Alte	velho
+Araujo-1996-289-123	123	Young Man	76.0	YOUNG MAN	Jüngling	moço
+Araujo-1996-289-124	124	Twelve Year-Old Boy			Knabe von 12 Jahren	garoto de doze anos
+Araujo-1996-289-125	125	Seven Year-Old Boy			Knabe von 7 Jahren	garoto de sete anos
+Araujo-1996-289-126	126	Small Boy			kleiner Knabe	pequeno garoto
+Araujo-1996-289-127	127	Nursling	1289.0	BABY	Säugling	lactente
+Araujo-1996-289-128	128	Woman	962.0	WOMAN	Frau	mulher
+Araujo-1996-289-129	129	Young Woman	77.0	YOUNG WOMAN	großes Mädchen	moça
+Araujo-1996-289-130	130	Twelve Year-Old Girl			Mädchen von 12 Jahren	menina de doze anos
+Araujo-1996-289-131	131	Girl	1646.0	GIRL	kleines Mädchen	menininha
+Araujo-1996-289-132	132	Father	1217.0	FATHER	Vater	pai
+Araujo-1996-289-133	133	Mother	1216.0	MOTHER	Mutter	mãe
+Araujo-1996-289-134	134	His Mother			seine Mutter	mãe dele
+Araujo-1996-289-135	135	My Wife			meine Frau	minha mulher
+Araujo-1996-289-136	136	White Person			Weißer	Branco
+Araujo-1996-289-137	137	Black Person			Neger	Negro
+Araujo-1996-289-138	138	Indigenous Peoples Of The Americas	3612.0	INDIGENOUS PEOPLES OF THE AMERICAS	Indianer	Índio
+Araujo-1996-289-139	139	Maxakalí			Mašakarí­	Maxakali
+Araujo-1996-289-140	140	Botocudo			Botocudo	Botocudo
+Araujo-1996-289-141	141	Pataxó			Patašó	Pataxó
+Araujo-1996-289-142	142	Men'S House	408.0	MEN'S HOUSE	Männerhaus	casa dos homens
+Araujo-1996-289-143	143	Dance Place			Tanzplatz	lugar de dança
+Araujo-1996-289-144	144	Dance Mask			Tanzmaske	máscara de dança
+Araujo-1996-289-145	145	Dance Mast			Tanzpfahl	mastro para dança
+Araujo-1996-289-146	146	Dancing Rattle	300.0	RATTLE	Tanzrassel	chocalho para dança
+Araujo-1996-289-147	147	Dancing Rammer (Olyra)			Stampfrohr	tubo [de taquara] para bater (o ritmo)
+Araujo-1996-289-148	148	Shade Or Shadow	1388.0	SHADE	Schatten	sombra
+Araujo-1996-289-149	149	Soul Of Dead Person	1175.0	GHOST	Totenseele	alma do morto
+Araujo-1996-289-150	150	Rainstick			Schwirrholz	(pau) zunidor
+Araujo-1996-289-151	151	Monkey	1350.0	MONKEY	Affe	macaco
+Araujo-1996-289-152	152	Howler Monkey	433.0	HOWLER MONKEY	Brüllaffe	guariba
+Araujo-1996-289-153	153	Bat	1793.0	BAT	Fledermaus	morcego
+Araujo-1996-289-154	154	Dog	2009.0	DOG	Hund	cachorro
+Araujo-1996-289-155	155	Giant Otter (Pteronura Brasiliensis)	15.0	OTTER	Fischotter	lontra, ariranha
+Araujo-1996-289-156	156	Jaguar	1250.0	JAGUAR	Yaguar	onça
+Araujo-1996-289-157	157	Black Jaguar			schwarzer Yaguar	onça preta
+Araujo-1996-289-158	158	Puma	1891.0	PUMA	Puma	puma
+Araujo-1996-289-159	159	Ocelot			Yaguatirica	jaguatirica
+Araujo-1996-289-160	160	Porcupine	2510.0	PORCUPINE	Stachelschwein	porco espinho
+Araujo-1996-289-161	161	Rabbit	1136.0	RABBIT	Hase	coelho
+Araujo-1996-289-162	162	Capybara	314.0	CAPYBARA	Capivara	capivara
+Araujo-1996-289-163	163	Paca	316.0	PACA	Paca	paca
+Araujo-1996-289-164	164	Agouti	313.0	AGOUTI	Cutia	cotia
+Araujo-1996-289-165	165	Ox	1169.0	OX	Rinde	boi
```

Comment:
German translation: "Rind" or "Ochse"

Reply:
So the German is in the original, right? If this is the case, @marthuis, can you check the spelling is what we find there? If so, we should leave it, but it could be a typing problem as well.